demo/src/main/java/com/example/demo/impl/IStudentDAO.java

```diff
@@ -0,0 +1,11 @@
+package com.example.demo.impl;
+
+import com.example.demo.dto.StudentDTO;
+import com.example.demo.entity.StudentEntity;
+
+public interface IStudentDAO {
+	   void saveStudentInfo(StudentDTO studentDTO);
```

Comment:
These methods should have return types depicting the status of the operation being performed.